frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -44,7 +44,7 @@ export const getPipelineRunData = (
   const params = latestRunParams || getPipelineRunParams(pipelineParams);
 
   return {
-    apiVersion: `${PipelineRunModel.apiGroup}/${PipelineRunModel.apiVersion}`,
+    apiVersion: pipeline ? pipeline.apiVersion : latestRun.apiVersion,
```

Comment:
Prep for changing apiVersions (unrelated to this fix though - future assistance)

Reply:
sorted